infrastructure/variables.tf

```diff
@@ -26,3 +26,11 @@ variable "application_type" {
   default     = "web"
   description = "Type of Application Insights (Web/Other)"
 }
+
+variable "redis_family" {
+  default     = "P"
+}
+
+variable "redis_capacity" {
+  default     = "2"
```

Comment:
TODO: Back down to `1` by default; currently set to `2` for verification on pipeline against AAT.

Reply:
Following discussion, will upgrade all envs instead.